package.json

```diff
@@ -55,6 +55,7 @@
     "merge-stream": "2.0.0",
     "pkg-dir": "4.2.0",
     "prismjs": ">=1.24.0",
+    "spec-up": "^0.10.6",
```

Comment:
Hmm…seems weird this is needed. I’ve not done it in a while, but have definitely tested locally and it had been working fine.  Doesn’t seem like it was ever removed.  And the GHA publishing is working fine...

Reply:
Kevin recently added the spec-up tooling.

Reply:
removed this in https://github.com/trustoverip/tswg-did-method-webs-specification/pull/117/commits/6cd709751e077208ecee9cdbff527438f3c9ff74